Makefile

```diff
@@ -705,6 +705,7 @@ out/doc/api/assets/%: doc/api_assets/% out/doc/api/assets
 
 
 run-npm-ci = $(PWD)/$(NPM) ci
+run-npm-task = $(PWD)/$(NPM)
```

Comment:
Might this need to be this?:

```
run-npm-task = $(PWD)/$(NPM) run
```

...so that it's `npm run lint` later on instead of `npm lint` which won't work?

Reply:
Yeap, you are right. I miss the run command.